op-conductor/conductor/service_test.go

```diff
@@ -751,7 +752,7 @@ func (s *OpConductorTestSuite) TestFailureAndRetry3() {
 			s.executeAction()
 		}
 		return res
-	}, 2*time.Second, 100*time.Millisecond)
+	}, 10*time.Millisecond, time.Millisecond)
```

Comment:
This might be a little too short, depending on how tests are executed, the main loop might've not been executed.

Suggesting to keep the max time higher, 2 seconds, etc